src/Tizen.NUI/src/public/Window.cs

```diff
@@ -92,6 +92,7 @@ public Window(Rectangle windowPosition = null , bool isTranslucent = false) : th
         [EditorBrowsable(EditorBrowsableState.Never)]
         public Window(string name, Rectangle windowPosition = null, bool isTranslucent = false) : this(Interop.Window.Window_New__SWIG_0(Rectangle.getCPtr(windowPosition), name, isTranslucent), true)
         {
+            this._windowTitle = name;
```

Comment:
it sets this._windowTitle which seems to be a private member variable.
then by this set, the name will be set to native Window?